### PR TITLE
passwordPoliciesHelp.notContainsUsername missing in admin console

### LINKED ISSUE #28798: `passwordPoliciesHelp.notContainsUsername` missing in admin console
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

`passwordPoliciesHelp.notContainsUsername` translation missing, related to #27643

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

Help text of `Not Contains Username` password policy is defined

### Actual behavior

Help text of `Not Contains Username` password policy is not defined

### How to Reproduce?

Go to admin console, then Authentication -> Policies and add _Not Contains Username_ policy, click question mark.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by jndev-42**
- Add passwordPoliciesHelp.notContainsUsername for messages_en/es/fr
- Add all passwordPoliciesHelp found in messages_en file for messages_fr
- Fix minor encoding issues in messages_es file

Closes #28798

### CODE REVIEW

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties [hunk 1]
```diff
@@ -56,7 +56,7 @@ credentialsList=Lista de credenciales
 usermodel.clientRoleMapping.clientId.label=ID de cliente
 clientId=ID de cliente
 serviceProviderEntityId=ID de entidad del proveedor de servicios
-internationalizationHelp=Si está habilitado, puedes elegir qué ubicaciones admites para este reino y cuál es la ubicaci�n predeterminada.
+internationalizationHelp=Si está habilitado, puedes elegir qué ubicaciones admites para este reino y cuál es la ubicaci�n predeterminada.
```
Comment (jonkoops) [inline js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties:59]: Can we remove the weird unicode here as well?
```suggestion
internationalizationHelp=Si está habilitado, puedes elegir qué ubicaciones admites para este reino y cuál es la ubicación predeterminada.
```

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties [hunk 2]
```diff
@@ -98,7 +98,7 @@ passwordPolicyHintsEnabledHelp=Aplicable solo para MSAD editable. Si está activ
 expirationValueNotValid=El valor debe ser mayor o igual a 1
 eventTypes.UPDATE_CONSENT.name=Actualizar consentimiento
 forceArtifactBinding=Forzar la vinculación de artefactos
-eventTypes.REFRESH_TOKEN_ERROR.description=Error de token de actualizaci�n
+eventTypes.REFRESH_TOKEN_ERROR.description=Error de token de actualizaci�n
```
Comment (jonkoops) [inline js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties:101]: Same here, only use valid UTF-8.
```suggestion
eventTypes.REFRESH_TOKEN_ERROR.description=Error de token de actualización
```

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties
Comment (jonkoops) [inline js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties:1]: Please get another native speaker to review this file.

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_fr.properties
Comment (jonkoops) [inline js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_fr.properties:1]: Please get another native speaker to review this file.

**Review by jonkoops — CHANGES_REQUESTED** [review]
Looks like some weird uni

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties (hunk 1 above)
Comment (Kizoku-Dev) [reply js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties]: Done, I apologise as i didn't see it

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties (hunk 2 above)
Comment (Kizoku-Dev) [reply js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties]: Done, I apologise as i didn't see it

Comment (jonkoops) [reply js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties]: No problem, you already removed a bunch, which is great!

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_fr.properties
Comment (MatthieuPeyrot) [reply js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_fr.properties:1]: Looks good 👍

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties
Comment (MatthieuPeyrot) [reply js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties:1]: Looks good too 👍

**Review by MatthieuPeyrot — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**